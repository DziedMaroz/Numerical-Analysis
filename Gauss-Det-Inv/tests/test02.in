6
1 2 0 0 0 0
3 8 0 0 0 0 
-2 3 2 -1 0 0
7 2 3 2 0 0
5 -1 3 5 7 -5
2 3 7 2 2 -1
