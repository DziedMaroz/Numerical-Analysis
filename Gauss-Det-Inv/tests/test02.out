RUNNING ON TEST: tests/test02.in
det (A) =   42.000
det(A)*A^-1:
  168.00  -42.00   -0.00   -0.00   -0.00    0.00
  -63.00   21.00    0.00    0.00    0.00   -0.00
   -0.00   -6.00   12.00    6.00   -0.00   -0.00
 -525.00  135.00  -18.00   12.00    0.00   -0.00
  931.00 -203.00  -98.00  -84.00  -14.00   70.00
  959.00 -199.00 -148.00 -102.00  -28.00   98.00
